testdata/resources/docker_service/testAccDockerServicePartialReplicationConfigStep3.tf

```diff
@@ -4,11 +4,17 @@ provider "docker" {
   }
 }
 
+resource "docker_image" "tftest_image" {
+  name         = "127.0.0.1:15000/tftest-service:v1"
+  keep_locally = false
+  force_remove = true
+}
+
 resource "docker_service" "foo" {
   name = "tftest-service-basic"
   task_spec {
     container_spec {
-      image             = "127.0.0.1:15000/tftest-service:v1"
+      image             = docker_image.tftest_image.latest
```

Comment:
```suggestion
      image             = docker_image.tftest_image.id
```

Reply:
see 58a5e8599beaabff8aedab2682e1e894dbaddc93 and 3a6790cf1eb69279e2a0b37ad49d8a2d89f2c5f2